code/modules/organs/prosthetics/prosthetics_manufacturer.dm

```diff
@@ -20,6 +20,14 @@
 	var/modular_prosthetic_tier = MODULAR_BODYPART_INVALID        // Determines how the limb behaves as a prosthetic with regards to manual attachment/detachment.
 	var/limb_tech = "{'engineering':1,'materials':1,'magnets':1}" // What tech levels should limbs of this type use/need?
 
+/decl/prosthetics_manufacturer/validate()
+	. = ..()
+	if(icon && (!applies_to_part || (BP_CHEST in applies_to_part)))
+		if(check_state_in_icon("torso", icon))
+			. += "torso state present in icon"
+		if(!check_state_in_icon(BP_CHEST, icon))
+			. += "chest state not present in icon"
```

Comment:
If for some reason we ever change what BP_CHEST is defined to, this should probably say the proper state name.
```suggestion
			. += BP_CHEST + " state not present in icon"
```